frontend/src/components/forms/CreateLog.tsx

```diff
@@ -389,6 +387,18 @@ const CreateLog = () => {
                       options={EMPLOYEES}
                       placeholder="Select Employee"
                       onChange={handleAttnToChange}
+                      styles={{
```

Comment:
Update the styles to only use selectStyle
```suggestion
                      styles={selectStyle}
```